reports/live-1-domains/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.17
```

Comment:
Can we use 1.18?

Reply:
yes should be no problem in changing to 1.18 I have been using this locally